README.md

```diff
@@ -1 +1,10 @@
-# ss-deploy
\ No newline at end of file
+# ss-deploy
+
+## Setup
+
+1. Copy the `.env.example` file to `.env`:
+2. Change the values in the `.env` file to your desired values
+3. Run `docker compose build`
+4. Run `docker-compose up -d`
+5. Open `http://localhost:{SS_WEB_PORT}}` in your browser
```

Comment:
I'm sure you checked it works on your end; my ports are messed up, would need to go over it with you at some point again :(
Also just noticed extra } at the end of url.

Reply:
yea we can take a look whenever you get a moment